example/image-classification/train_imagenet.py

```diff
@@ -31,7 +31,7 @@
     data.add_data_args(parser)
     data.add_data_aug_args(parser)
     # use a large aug level
-    data.set_data_aug_level(parser, 3)
+    # data.set_data_aug_level(parser, 3)
```

Comment:
?

Reply:
The `set_data_aug_level` doesn't set standard imagenet training augmentation.